heron/schedulers/src/java/org/apache/heron/scheduler/kubernetes/V1Controller.java

```diff
@@ -1101,80 +1104,212 @@ protected V1ConfigMap getConfigMap(String configMapName) {
   }
 
   /**
-   * Generates the <code>Volume</code> and <code>Volume Mounts</code> to be placed in the <code>executor container</code>.
-   * @param mapConfig Mapping of <code>Volumes</code> to <code>key-value</code> configuration pairs.
-   * @return A pair of configured lists of <code>V1Volume</code> and <code>V1VolumeMount</code>.
+   * Generates the <code>Volume Mounts</code> to be placed in the <code>Executor</code>
+   * and <code>Manager</code> from options on the CLI.
+   * @param volumeName Name of the <code>Volume</code>.
+   * @param configs Mapping of <code>Volume</code> option <code>key-value</code> configuration pairs.
+   * @return A configured <code>V1VolumeMount</code>.
+   */
+  @VisibleForTesting
+  protected V1VolumeMount createVolumeMountsCLI(String volumeName,
```

Comment:
Would it be worthwhile to mark this parameters passed in the function as `final`?

Reply:
Changes affected.